package.json

```diff
@@ -55,5 +55,11 @@
     "mocha": "^2.4.5",
     "rx": "^4.0.7",
     "uglifyify": "^3.0.1"
+  },
+  "peerDependencies": {
```

Comment:
Hiya! Can we please make xstream a dependency instead of a peerDependency? As far as I know, peerDependencies are deprecated in NPM 3.